src/Model/Event.php

```diff
@@ -86,4 +86,19 @@ public static function getName(
 
         return "{$moment->value}.{$name}";
     }
+
+    public function getUseCaseRequest(): mixed
+    {
+        return $this->parameters['useCaseRequest'];
+    }
+
+    public function getUseCaseResponse(): mixed
+    {
+        return $this->response;
+    }
+
+    public function getUseCaseException(): mixed
+    {
+        return $this->exception;
+    }
```

Comment:
@sidux Sorry, I add this to the PR too.
These changes should be merged too for the same reason.
Are you ok ?